packages/react-components/react-tags/src/components/Tag/useTagStyles.ts

```diff
@@ -1,20 +1,46 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { TagSlots, TagState } from './Tag.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const tagClassNames: SlotClassNames<TagSlots> = {
   root: 'fui-Tag',
-  // TODO: add class names for all slots on TagSlots.
-  // Should be of the form `<slotName>: 'fui-Tag__<slotName>`
+  content: 'fui-Tag__content',
+  avatar: 'fui-Tag__avatar',
+  icon: 'fui-Tag__icon',
+  primaryText: 'fui-Tag__primaryText',
+  secondaryText: 'fui-Tag__secondaryText',
+  dismissButton: 'fui-Tag__dismissButton',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'inline-flex',
   },
+  content: {
+    display: 'inline-grid',
+    gridTemplateColumns: 'auto 8px auto auto 8px auto',
+    gridTemplateRows: '1fr auto auto 1fr',
+    gridTemplateAreas: `
+    "avatar x icon .         y"
+    "avatar x icon primary   y"
+    "avatar x icon secondary y"
+    "avatar x icon .         y"
```

Comment:
as I understand x is not assigned to anything, right? and the use of it is to have a gap between the avatar and icon. If that's the case, you can remove the x and leave dots:

```suggestion
    "avatar . icon .         y"
    "avatar . icon primary   y"
    "avatar . icon secondary y"
    "avatar . icon .         y"
```